msu/systems/mod_settings/settings_element.nut

```diff
@@ -40,6 +40,11 @@
 		return this.getPage().getPanelID();
 	}
 
+	function getModID()
+	{
+		return this.getPage().getPanel().getModID();
```

Comment:
Shouldn't this be simplifies to `this.getPage.getModID()`?

Reply:
Will do